test/system/160-volumes.bats

```diff
@@ -327,11 +327,22 @@ EOF
     mkdir $myvoldir
     touch $myvoldir/myfile
 
+    containersconf=${PODMAN_TMPDIR}/containers.conf
+    cat >$containersconf <<EOF
+[containers]
+userns="keep-id"
+EOF
+
     # With keep-id
     run_podman run --rm -v $myvoldir:/vol:z --userns=keep-id $IMAGE \
                stat -c "%u:%s" /vol/myfile
     is "$output" "$(id -u):0" "with keep-id: stat(file in container) == my uid"
 
+    # With keep-id from containers.conf
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run --rm -v $myvoldir:/vol:z --userns=keep-id $IMAGE \
+               stat -c "%u:%s" /vol/myfile
+    is "$output" "$(id -u):0" "with keep-id: stat(file in container) == my uid"
```

Comment:
should have a test description different from the previous one so that a log reader can know which of the checks failed.